manifests/platforms/aks.jsonnet

```diff
@@ -99,6 +99,10 @@ local grafana = import "../components/grafana.jsonnet";
       data_+: $.config.oauthProxy,
     },
 
+    ingress+: {
```

Comment:
Can't we have this in a generic jsonnet file instead of three times (for each platform-specific jsonnet)?

Reply:
Agreed, we should refactor the common parts out of all platforms into a platforms/common.libsonnet or something.  I think originally I expected greater deviation between platforms.

I would rather do that refactor in a separate PR, after this PR.  I can do that before (or within this PR) if you would prefer.